docs/README.md

```diff
@@ -75,6 +108,229 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
 
 **Note:** The `IstioOperator` / `istioctl` example is separate from the Sail-operator. Setting `spec.components` or `spec.values.gateways` on your Sail-operator `Istio` resource **will not work**.
 
+## Update Strategy
+
+The sail-operator supports two update strategies to update the version of the Istio control plane: `InPlace` and `RevisionBased`. The default strategy is `InPlace`.
+
+### InPlace
+When the `InPlace` strategy is used, the existing Istio control plane is updated in-place. The workloads therefore don't need to be moved from one control plane instance to another.
+
+#### Example using the InPlace strategy
+
+Prerequisites:
+* sail-operator needs to be installed.
+* Deploy `IstioCni` resource.
```

Comment:
IstioCNI is not a prerequisite. Since these aren't OpenShift-specific docs, we can skip IstioCNI completely.